rewards/protobuf/rewards.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+// Rewards - This is the gRPC definition of the rewards service
+service Rewards {
+    // GetParameters - This RPC call will retrieve the
+    // valid rewards parameters from the service and return
+    // a parameters response instance
+    rpc GetParameters(ParametersRequest) returns(ParametersResponse) {}
+}
+
+enum Currency {
```

Comment:
I feel like there are more currencies in the `BaseCurrency` library. What's the difference between these two?

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/407/commits/2db23eb4cc994da49259718432fffc43f2808d41